data/pools/transactionPool.go

```diff
@@ -694,6 +694,7 @@ func (pool *TransactionPool) recomputeBlockEvaluator(committedTxIds map[transact
 	pool.assemblyMu.Unlock()
 
 	next := bookkeeping.MakeBlock(prev)
+	proto := config.Consensus[next.CurrentProtocol]
```

Comment:
nit: this can move down to line 719. No need for the map lookup until later.

Reply:
No longer necessary since I moved away from using `MaxTxnLife` thanks to your other comment